server/templates/assignment_details_email.html

```diff
@@ -0,0 +1,59 @@
+<b>Slugline: </b>{{ slugline }}
+<p>
+    <b>Description: </b>{{ assignment.description_text }}
+</p>
+{% if event %}
+<p>
+    <b>Event: </b>{{ event.name }}<br>
+    <b>Start: </b>{{ event.dates.start|format_datetime(timezone_string=event.dates.tz) }}
```

Comment:
Would the end date/time be useful to reporters as well?

Reply:
Maybe,  I was just going on the product owners requirements!

Reply:
No you weren't. "(Event date/time)" is what was specified.